study.md

```diff
@@ -33,7 +34,10 @@ How does Ember use the URL to load view-state? Which layers in Ember are
 responsible for which tasks?
 
 ```md
-<!-- your answer here -->
+Ember uses four options to manage URLs being history, hash, auto, and none.
+History uses the HTML history api to produce URLs. The hash option loads the
+beginning state of the app and keeps it in a current state. This is what Ember
+uses to load current view-state.
```

Comment:
All correct. `location` is set in the Router, the part of the Ember app that's responsible for translating a URL string to loading a particular route and showing that route's view state.